### PR TITLE
Navigation links on home page

### LINKED ISSUE #6: Pull Data: Navigation

### PR DESCRIPTION
**Pull request by jean-zhang**
### Note: A bunch of changes made on Prismic
- Navigation is now a list of links so she can add and remove as she likes
- Navigation supports both Page (Document) links and Web links (so if she wants to move to a more complex google form etc)
- Pages now have UIDs that represent their slug to support the above behavior

### Added to the functions within api/richText.js
- now have function that grabs just the text without styling

Screen Shot 2021-03-16 at 10 16 46 AM

### CODE REVIEW

File: components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { PageLink } from "../lib/link";
+import { getString } from "../lib/richText";
+
+export default function Navigation({ navigationData }) {
+  const { navigation_links: navigationLinks } = navigationData;
+  console.log(navigationData);
```
Comment (jzhzhang66) [inline components/Navigation.js]: I think you forgot to delete this console.log - otherwise, looks good!

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
I think you forgot to delete the console.log on line 7 of Navigation.js. Otherwise, looks good!

File: pages/index.js [hunk 2]
```diff
@@ -12,6 +13,7 @@ export default function HomePage({ homePageData }) {
 
   return (
     <>
+      <Navigation navigationData={navigationData} />
```
Comment (josephmannis) [inline pages/index.js:16]: I assume you want this to show up on every page – you can either put it in _app.js or [document.js](https://nextjs.org/docs/advanced-features/custom-document). If not, this is fine!

**Review by josephmannis — COMMENTED** [review]
Just a check about the nav thing. Please update it if you need nav on every page! Otherwise, all good ☀️

File: pages/index.js (hunk 2 above)
Comment (jean-zhang) [reply pages/index.js:16]: Based on the documentation seems like it _app.js and _document.js don't support Data Fetching methods like getStaticProps or getServerSideProps.

**Pull request merged**